onnxmltools/convert/sklearn/operator_converters/NaiveBayes.py

```diff
@@ -100,7 +104,35 @@ def convert_sklearn_naive_bayes(scope, operator, container):
     #                       reshaped_result[M,]             --------------------------------------RESHAPE
     #                                   |                   |
     #                                   V                   |
-    #  onnx_proto.TensorProto.INT64 -> CAST -> output[M,] <-|
+    # (to=onnx_proto.TensorProto.INT64)CAST -> output[M,] <-|
+    #
+    #
+    # If model's binarize attribute is not null, then input is updated based on the following graph:
```

Comment:
Input can't not be upated because ONNX is single-static-assignment, which mean cycle in graph is not allowed. I guess you probably want to say `The input of BN (and multi-class BN) is produced by the following pre-processing graph`.